Cargo.toml

```diff
@@ -7,32 +9,39 @@ members = [
 	"clap_mangen",
 ]
 
+[workspace.package]
+categories = ["command-line-interface"]
```

Comment:
I feel like `categories` isn't a great form of sharing as it doesn't encourage a new crate author to consider what categories are appropriate but just blindly use the workspace.  Kind of like why not to shared `keywords`.

Reply:
I was worried about this and honestly was wondering what to do. It almost seems like you should be able to inherit and specify extras but I don't love that idea